### PR TITLE
Admin component preview readme file

### PR DESCRIPTION
**Pull request by vandebled**

### CODE REVIEW

File: Features/AdminComponentPreview/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Admin Component Preview (Flynt Feature)
+
+Enables component's to be previewed in multiple scenarios:
+
+## Front-end
+
+Add a full width layout screenshot of a module to its module folder to show a preview in the Wordpress backend.
```
Comment (emcarru) [inline Features/AdminComponentPreview/README.md]: component not module

File: Features/AdminComponentPreview/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+When logged in, you can overlay the screenshots with opacity in the frontend for easy pixel perfect development by clicking the'Component Preview' button in the WP Admin Bar, drag the images around and move each image with the arrow keys.
```
Comment (emcarru) [inline Features/AdminComponentPreview/README.md]: `the 'Component` <- missing space

File: Features/AdminComponentPreview/README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+## Back-end
```
Comment (emcarru) [inline Features/AdminComponentPreview/README.md]: Back End

File: Features/AdminComponentPreview/README.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# Admin Component Preview (Flynt Feature)
+
+Enables component's to be previewed in multiple scenarios:
+
+## Front-end
```
Comment (emcarru) [inline Features/AdminComponentPreview/README.md]: Front End

File: Features/AdminComponentPreview/README.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+Add a full width layout screenshot of a module to its module folder to show a preview in the Wordpress backend.
+
+When logged in, you can overlay the screenshots with opacity in the frontend for easy pixel perfect development by clicking the'Component Preview' button in the WP Admin Bar, drag the images around and move each image with the arrow keys.
+
+## Back-end
+
+The preview image will be shown beside the component name in the flexible content group layout.
+
+Orollover on a component name inside the "Add Component" dropdown menu, it will show the desktop and mobile preview images.
```
Comment (emcarru) [inline Features/AdminComponentPreview/README.md]: Orollover?

File: Features/AdminComponentPreview/README.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
+# Admin Component Preview (Flynt Feature)
+
+Enables component's to be previewed in multiple scenarios:
+
+## Front end
+
+Add a full width layout screenshot of a component to its component folder to show a preview in the Wordpress backend.
```
Comment (emcarru) [inline Features/AdminComponentPreview/README.md]: WordPress

File: Features/AdminComponentPreview/README.md [hunk 7]
```diff
@@ -0,0 +1,22 @@
+# Admin Component Preview (Flynt Feature)
+
+Enables components to be previewed in multiple scenarios:
+
+## Front end
+
+Add a full width layout screenshot of a component to its component folder to show a preview in the Wordpress backend.
+
+When logged in, you can overlay the screenshots with opacity in the frontend for easy pixel perfect development by clicking the 'Component Preview' button in the WP Admin Bar, drag the images around and move each image with the arrow keys.
```
Comment (emcarru) [inline Features/AdminComponentPreview/README.md]: Can make it less passive:

`When logged in, click the 'Component Preview' button on the WordPress admin bar to overlay a preview image (with reduced opacity). For easy pixel perfect development, drag the images around or move them with the arrow keys.

File: Features/AdminComponentPreview/README.md [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+
+## Front end
+
+When logged in, you can overlay the screenshots with opacity in the frontend for easy pixel perfect development by clicking the 'Component Preview' button in the WordPress Admin Bar, drag the images around and move each image with the arrow keys.
+
+## Back end
+
+The preview image will be shown beside the component name in the flexible content group layout.
+
+On rollover on a component name inside the "Add Component" dropdown menu, it will show the desktop and mobile preview images.
```
Comment (emcarru) [inline Features/AdminComponentPreview/README.md]: Again would avoid any passive tense:

'Rollover a component name inside the "Add Component" dropdown menu to see the desktop and mobile preview images."

File: Features/AdminComponentPreview/README.md [hunk 9]
```diff
@@ -0,0 +1,22 @@
+# Admin Component Preview (Flynt Feature)
+
+Add a full width layout screenshot of a component to its component folder to show a preview in the Wordpress back end and front end.
+
+Components previewed in multiple scenarios:
```
Comment (emcarru) [inline Features/AdminComponentPreview/README.md]: Sorry, last adjustments:

"Add layout screenshots of a component to its component folder to show a preview in the front end and WordPress back end.

Components can then be previewed in multiple scenarios:"

File: Features/AdminComponentPreview/README.md [hunk 10]
```diff
@@ -0,0 +1,22 @@
...
+The preview image will be shown beside the component name in the flexible content group layout.
+
+Rollover a component name inside the "Add Component" dropdown menu to see the desktop and mobile preview images.
+
+## Usage
+
+Prepare a file size optimized  `preview-desktop.jpg` and `preview-mobile.jpg` for your component.
+
+- `preview-desktop.jpg` should be 1440x900 px.
+- `preview-mobile.jpg` should be 320x480 px.
```
Comment (Qakulukiam) [inline Features/AdminComponentPreview/README.md]: not sure if it's obvious, but should we mention that these files need to be in the components folder?

Comment (vandebled) [inline Features/AdminComponentPreview/README.md]: It's mentioned in the first sentence of the readme, but i can specify it here too

**Pull request merged**